src/core/server/integration_tests/http/versioned_router.test.ts

```diff
@@ -137,11 +135,9 @@ describe('Routing versioned requests', () => {
         .set('Elastic-Api-Version', 'abc')
         .expect(400)
         .then(({ body }) => body)
-    ).resolves.toEqual(
-      expect.objectContaining({
-        message: expect.stringMatching(/Invalid version/),
-      })
-    );
+    ).resolves.toMatchObject({
+      message: expect.stringMatching(/Invalid version/),
+    });
```

Comment:
Replaced use of `toEqual(expect.objectContaining({...` for more terse `toMatchObject({...` API